Project_2/README.md

```diff
@@ -1,16 +1,76 @@
 # GroupAwesome-AwesomeProject1
 
-Some of the problem i had while trying to make this for java:
-That the question makes me print things from the consumer and the job, which means I can't even use interfaces.
+## 1. Accomplished Items  
```

Comment:
We are missing the design part of the read me. I would add it but I'm to lazy to make a branch and PR, so here you go. 

Java:
When dealing with monitors (or class wide) locks, we can not gave the consumer notify the producer as java does not differentiate between different locks in different functions. 
The design is based on a standard producer and consumer pattern with a monitor queue. When a queue is empty any consumer attempting to pull from the queue will wait for a notification from the producer. When the queue is full and the producer tries to add to the queue it will release its lock and sleep for one second. This number can be changed in the Producer.java file.

C:
The design is based on a standard producer and consumer pattern with a blocking queue. When a queue is empty any consumer attempting to pull from the queue it will be stopped by a semaphore and have to wait for a notification from the producer. When a queue is full and the producer attempts to put a task in the queue it will be stopped by a semaphore and have to wait for a notification from any consumer. To enforce concurrency there is a mutex that is used during add and remove process of the queue, this ensures that only one thread can add or remove items from the queue at one time.

Reply:
Added

Reply:
How do I resolve this. How do I please the Github overlords. I have made the changes. Please let me resolve this. :(